SitePackage.lua

```diff
@@ -132,6 +132,26 @@ local function msg_hook(mode, output)
 
     dbg.print{"Mode is ", mode, "\n"}
 
+    if output[1] and output[1]:find("Your site prevents the automatic swapping of modules with same name") then
+        -- find the module name causing the issue (almost always toolchain module)
+        local mname
+        mname = output[1]:match("$ module swap [^ ]+%s+([^ \n]+)")
+        local mStack   = ModuleStack:moduleStack()
+
+        local label  = colorize("red", "Lmod has detected the following error: ")
+        local twidth = TermWidth()
+        local s      = {}
+        s[#s+1] = buildMsg(twidth, label, "You can not load modules using different toolchains.",
```

Comment:
@wpoely86 this error will not only pop up when two versions of the same toolchain but different version is loaded, but also for other conflicts on module name, right?

For example, if someone loads A which depends on Python 2.7.12, and then loads B which depends on Python 3.5.2?

So, we should not mention 'toolchian' at all in the error message, and only talk about 'module name' and 'module version'?